wfe/wfe_test.go

```diff
@@ -588,10 +593,12 @@ func TestDirectory(t *testing.T) {
 	})
 	test.AssertEquals(t, responseWriter.Header().Get("Content-Type"), "application/json")
 	test.AssertEquals(t, responseWriter.Code, http.StatusOK)
-	assertJSONEquals(t, responseWriter.Body.String(), `{"new-authz":"http://localhost:4300/acme/new-authz","new-cert":"http://localhost:4300/acme/new-cert","new-reg":"http://localhost:4300/acme/new-reg","revoke-cert":"http://localhost:4300/acme/revoke-cert"}`)
+	assertJSONEquals(t, responseWriter.Body.String(), `{"key-change":"http://localhost:4300/acme/key-change","new-authz":"http://localhost:4300/acme/new-authz","new-cert":"http://localhost:4300/acme/new-cert","new-reg":"http://localhost:4300/acme/new-reg","revoke-cert":"http://localhost:4300/acme/revoke-cert"}`)
```

Comment:
Looking at the Certbot commit history, https://github.com/certbot/certbot/commit/6f9e28fccad96c19b3fd58560d5c6ddba8201d04 was the commit that fixed the "brittle directory" problem, and was first included in the `v0.6.0` tag. Can you calculate what percentage of new-cert requests today are coming from pre-0.6.0 certbot clients? We may need to provide this via an alternate directory.